stripe-core/src/main/java/com/stripe/android/core/strings/ResolvableStringUtils.kt

```diff
@@ -14,49 +14,49 @@ internal fun resolveArgs(context: Context, args: List<Any?>): Array<Any?> {
 }
 
 /**
- * Creates a `ResolvableString` from an identifier.
+ * Creates a [ResolvableString] from an identifier.
  *
  * @param id a string resource identifier
  *
- * @return a `ResolvableString` instance when resolved returns the string that matches the identifier.
+ * @return a [ResolvableString] instance when resolved returns the string that matches the identifier.
  */
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 fun resolvableString(@StringRes id: Int): ResolvableString {
     return IdentifierResolvableString(id, listOf())
 }
 
 /**
- * Creates a `ResolvableString` from a given identifier.
+ * Creates a [ResolvableString] from a given identifier.
  *
  * @param id a string resource identifier
  * @param formatArgs a variable amount of arguments to format the string with.
  *
- * @return a `ResolvableString` instance when resolved returns the string that matches the identifier.
+ * @return a [ResolvableString] instance when resolved returns the string that matches the identifier.
  */
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 fun resolvableString(@StringRes id: Int, vararg formatArgs: Any?): ResolvableString {
```

Comment:
Actually, can we merge the id-only function and this one? Since we’re using `vararg`, this should be possible.